core/domain/suggestion_services.py

```diff
@@ -1012,6 +1015,13 @@ def get_reviewable_translation_suggestions_by_offset(
         user_id)
     language_codes = (
         contribution_rights.can_review_translation_for_language_codes)
+
+    if language is not None and language != '':
```

Comment:
Is this check necessary? If either of these conditions is true, then the language wouldn't exist in `language_codes`, right? `user_services.get_user_contribution_rights(
        user_id)` does validate that the language codes are valid, so I think we can remove this check.

Reply:
I see what you mean, makes sense. However, the result of this if is [language] or [], so I still need to make the empty check, because empty means that the caller wants all the languages. I am sending a commit with a reduced if statement.

Reply:
Commits sent, thanks!